examples/basic_mysql_connection/index.js

```diff
@@ -13,8 +13,23 @@ var connection = mysql.createConnection({
 
 connection.connect();
 
-connection.query('SELECT * FROM todos', function (error, results, fields) {
-    console.log(results);
-});
+let query=[	'select count(*) as items from todos',
+			'select * from todos where done',
+			'select * from todos where !done',
+			'select * from todos order by due DESC',
```

Comment:
This will retrieve the todos ordered by when they're due, but not when they're added. Sorry if the question wasn't clear, I'll work on that 😀